packages/amplify_datastore_plugin_interface/test/amplify_modelschema_test.dart

```diff
@@ -194,4 +194,63 @@ void main() {
           ])
     ]);
   });
+
+  test(
+      'Generated dart class Person should provide correct schema nests CustomType',
```

Comment:
Nit: Not totally sure if this is what you were going for but maybe
```suggestion
      'Generated dart class Person should provide correct schema with nested CustomType',
```